config/environments/production.rb

```diff
@@ -24,7 +24,11 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
This is only applied if the asset was not found. Maybe we don't need it, but I just copied what we do in Portus 😄